stix_shifter_modules/nozomi/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,555 @@
+{
+  "ip_src": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "src_ip"
+    },
+    {
+      "key": "ipv6-addr.value",
+      "object": "src_ip"
+    },
+    {
+      "key": "network-traffic.src_ref",
+      "object": "nt",
+      "references": "src_ip"
+    },
+    {
+      "key": "x-ibm-finding.src_ip_ref",
+      "object": "alert",
+      "references": "src_ip"
+    }
+  ],
+  "ip_dst": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "dst_ip"
+    },
+    {
+      "key": "ipv6-addr.value",
+      "object": "dst_ip"
```

Comment:
same problem as ip_src

Reply:
The stix-shifter framework does check internally for the ipv4-addr & ipv6-addr pattern matches. Hence transformer is not required.